reporter-web-app/build.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id "com.moowork.node" version "1.2.0"
+}
+
+node {
+    version = "8.11.4"
+    yarnVersion = "1.9.4"
```

Comment:
Should this be in any way aligned with the version in `Yarn.kt`? Even if not technically required it might look odd if we use another Yarn version ourselves that the one we expect others to use.

Reply:
I would rather update the required version in `Yarn.kt` in a separate PR than using an outdated version here.

Reply:
Agreed.